python/modules/IcePy/Types.cpp

```diff
@@ -42,6 +42,8 @@ static ExceptionInfoMap _exceptionInfoMap;
 namespace
 {
 
+const char emptySeq[0] = {};
```

Comment:
This code doesn't compile with `-pedantic` as it's not strict ANSI code... while this is probably ok if the extension isn't built with pedantic, we could instead just create an empty buffer in `createSequenceFromMemory` if `size == 0`, something like:
```
        // NOTE: PyMemoryView_FromMemory doesn't allow providing a null buffer pointer so
        // we manually construct an empty buffer here.
        Py_buffer pybuffer;
        if(PyBuffer_FillInfo(&pybuffer, 0, 0, 0, 1, PyBUF_SIMPLE) != 0)
        {
            assert(PyErr_Occurred());
            throw AbortMarshaling();
        }
        memoryview = PyMemoryView_FromBuffer(&pybuffer);
```